game-core/src/test/java/games/strategy/engine/lobby/client/login/LobbyPropertyFileParserTest.java

```diff
@@ -82,7 +74,7 @@ private static File newTempFile(final TestProps... testProps) throws Exception {
    */
   @Test
   void checkVersionSelection() throws Exception {
-    final File testFile = newTempFile(testDataSet());
+    final String testFile = newYaml(testDataSet());
```

Comment:
Probably want to rename this local since it's no longer a file (as was done above).